systemtests/src/main/java/com/github/streamshub/systemtests/utils/WaitUtils.java

```diff
@@ -0,0 +1,28 @@
+package com.github.streamshub.systemtests.utils;
+
+import io.fabric8.kubernetes.api.model.apps.Deployment;
+import io.skodjob.testframe.TestFrameConstants;
+import io.skodjob.testframe.wait.Wait;
+
+import static com.github.streamshub.systemtests.utils.ResourceUtils.listKubeResourcesByPrefix;
+
+public class WaitUtils {
+
+    private WaitUtils() {}
+
+    // ------------
+    // Deployment
+    // ------------
+    public static void waitForDeploymentWithPrefixIsReady(String namespaceName, String deploymentNamePrefix) {
+        Wait.until(String.format("creation of Deployment with prefix: %s in Namespace: %s", deploymentNamePrefix, namespaceName),
```

Comment:
Wait until + this string, may seem misleading -> combined with the cause, but for logging in `Wait.until` there is used a String ("Waiting for: " + cause)

Reply:
What is the advantage of `io.skodjob.testframe.wait.Wait` versus Awaitility?

Reply:
I think i went through the basic differences and i'd say the way of logging, onTimeout runnables and exceptions description are better with `Wait`. However you know more about the Awaitility then i do. Would you say that it is better? 

For example what about failing with custom messages that will ease the debugging - like "Failed waiting for Pod {} to roll in namespace {}"?

Reply:
Discussed offline - decided to keep `Wait.until` as it's a slightly more fitting the needs here.